### PR TITLE
Save GDT20 - Polar Stereographic

### PR DESCRIPTION
**Pull request by DPeterK**
Add save support for Grid Definition Template 20 - Polar Stereographic.

### CODE REVIEW

File: iris_grib/tests/integration/save_rules/test_grid_definition_section.py [hunk 1]
```diff
@@ -27,14 +27,13 @@
...
 
-from iris.coords import DimCoord
 from iris.coord_systems import (GeogCS,
                                 RotatedGeogCS,
                                 Mercator,
                                 TransverseMercator,
+                                Stereographic,
                                 LambertConformal,
                                 AlbersEqualArea)
-from iris.cube import Cube
```
Comment (dkillick) [inline iris_grib/tests/integration/save_rules/test_grid_definition_section.py]: Unused imports...

File: iris_grib/_save_rules.py [hunk 2]
```diff
@@ -634,17 +633,63 @@ def grid_definition_template_30(cube, grib):
...
+
+    # Retrieve the cube coord system.
+    cs = cube.coord(dimensions=[0]).coord_system
+
+    _perspective_projection_common(cube, grib)
+
+    # Is this a north or south polar stereographic projection?
+    if cs.central_lat == -90.0:
+        # XXX: is this the correct value? We read bit 0x80 in load, but set bit
+        #      0x1 when saving GDT30...
```
Comment (corinnebosley) [inline iris_grib/_save_rules.py]: I don't know, is it the correct value? This is a very unnerving comment.

**Comment by pp-mo** [comment]
I'm not happy with this as-is, 'cos I think we have some outstanding problems + we want some more real(er) testing.

Given the bug uncovered [above](https://github.com/SciTools/iris-grib/pull/122#discussion_r195332221), I'm thinking that at least one of NorthPolar/SouthPolar would currently fail a save-then-load roundtrip check. 
That's a good reason for having one.

File: iris_grib/_save_rules.py (hunk 2 above)
Comment (dkillick) [reply iris_grib/_save_rules.py]: > This is a very unnerving comment

Yup, it is. I assumed that it should be `0x80` but it was a bit odd that the GDT30 saver did not set that value...

Comment (pp-mo) [reply iris_grib/_save_rules.py]: > it was ***a bit odd*** that the GDT30 saver did not set that value

Bugs in existing implementation -- doesn't sound that unusual to me :rofl:

**Comment by bjlittle** [comment]
@DPeterK Fancy re-basing? :smile:

**Comment by DPeterK** [comment]
@bjlittle on it!

**Comment by bjlittle** [comment]
@pp-mo Do you fancy taking this PR on, and getting across the finish line? 
:car: :checkered_flag:

**Pull request closed without merging**